packages/schemas/src/foundations/jsonb-types.ts

```diff
@@ -1,4 +1,4 @@
-import { Language } from '@logto/phrases';
+import { Language } from '@logto/phrases/lib/types.js';
```

Comment:
this is used for SIE language - maybe we need change the type here to `string` and handle it in UI?

Reply:
issue created

Reply:
will do it in #1247, may be we should extract the phrase as a separate package so it can be imported by the schema package.